web/api/badges/web_buffer_svg.c

```diff
@@ -139,135 +139,7 @@ static double verdana11_widths[256] = {
     [124] = 4.9951171875, // |
     [125] = 6.982421875, // }
     [126] = 9.001953125, // ~
-    [127] = 0.0,
-    [128] = 0.0,
-    [129] = 0.0,
-    [130] = 0.0,
-    [131] = 0.0,
-    [132] = 0.0,
-    [133] = 0.0,
-    [134] = 0.0,
-    [135] = 0.0,
-    [136] = 0.0,
-    [137] = 0.0,
-    [138] = 0.0,
-    [139] = 0.0,
-    [140] = 0.0,
-    [141] = 0.0,
-    [142] = 0.0,
-    [143] = 0.0,
-    [144] = 0.0,
-    [145] = 0.0,
-    [146] = 0.0,
-    [147] = 0.0,
-    [148] = 0.0,
-    [149] = 0.0,
-    [150] = 0.0,
-    [151] = 0.0,
-    [152] = 0.0,
-    [153] = 0.0,
-    [154] = 0.0,
-    [155] = 0.0,
-    [156] = 0.0,
-    [157] = 0.0,
-    [158] = 0.0,
-    [159] = 0.0,
-    [160] = 0.0,
-    [161] = 0.0,
-    [162] = 0.0,
-    [163] = 0.0,
-    [164] = 0.0,
-    [165] = 0.0,
-    [166] = 0.0,
-    [167] = 0.0,
-    [168] = 0.0,
-    [169] = 0.0,
-    [170] = 0.0,
-    [171] = 0.0,
-    [172] = 0.0,
-    [173] = 0.0,
-    [174] = 0.0,
-    [175] = 0.0,
-    [176] = 0.0,
-    [177] = 0.0,
-    [178] = 0.0,
-    [179] = 0.0,
-    [180] = 0.0,
-    [181] = 0.0,
-    [182] = 0.0,
-    [183] = 0.0,
-    [184] = 0.0,
-    [185] = 0.0,
-    [186] = 0.0,
-    [187] = 0.0,
-    [188] = 0.0,
-    [189] = 0.0,
-    [190] = 0.0,
-    [191] = 0.0,
-    [192] = 0.0,
-    [193] = 0.0,
-    [194] = 0.0,
-    [195] = 0.0,
-    [196] = 0.0,
-    [197] = 0.0,
-    [198] = 0.0,
-    [199] = 0.0,
-    [200] = 0.0,
-    [201] = 0.0,
-    [202] = 0.0,
-    [203] = 0.0,
-    [204] = 0.0,
-    [205] = 0.0,
-    [206] = 0.0,
-    [207] = 0.0,
-    [208] = 0.0,
-    [209] = 0.0,
-    [210] = 0.0,
-    [211] = 0.0,
-    [212] = 0.0,
-    [213] = 0.0,
-    [214] = 0.0,
-    [215] = 0.0,
-    [216] = 0.0,
-    [217] = 0.0,
-    [218] = 0.0,
-    [219] = 0.0,
-    [220] = 0.0,
-    [221] = 0.0,
-    [222] = 0.0,
-    [223] = 0.0,
-    [224] = 0.0,
-    [225] = 0.0,
-    [226] = 0.0,
-    [227] = 0.0,
-    [228] = 0.0,
-    [229] = 0.0,
-    [230] = 0.0,
-    [231] = 0.0,
-    [232] = 0.0,
-    [233] = 0.0,
-    [234] = 0.0,
-    [235] = 0.0,
-    [236] = 0.0,
-    [237] = 0.0,
-    [238] = 0.0,
-    [239] = 0.0,
-    [240] = 0.0,
-    [241] = 0.0,
-    [242] = 0.0,
-    [243] = 0.0,
-    [244] = 0.0,
-    [245] = 0.0,
-    [246] = 0.0,
-    [247] = 0.0,
-    [248] = 0.0,
-    [249] = 0.0,
-    [250] = 0.0,
-    [251] = 0.0,
-    [252] = 0.0,
-    [253] = 0.0,
-    [254] = 0.0,
-    [255] = 0.0
+    [127] = 0.0
```

Comment:
It may be irrelevant to the change here, but there is a test that behaves differently now. I passed label=%82 (extended ASCII). I used to get spaces/empty chars, no I get 82. not sure why this happens though, please investigate.

Reply:
Well i removed it because we know only sizes of ASCII characters. Everything above 0x80 is UTF char and for it we guess size of 1em now (which will be updated by browser with actual rendered text size). Therefore I saw no use of values above 0x80 as we will never access them anyway